src/NServiceBus.AcceptanceTests/Core/FakeTransport/ProcessingOptimizations/When_using_concurrency_limit.cs

```diff
@@ -36,14 +37,21 @@ public ThrottledEndpoint()
 
         class FakeReceiver : IPushMessages
         {
+            PushSettings pushSettings;
+
             public Task Init(Func<MessageContext, Task> onMessage, Func<ErrorContext, Task<ErrorHandleResult>> onError, CriticalError criticalError, PushSettings settings)
             {
+                pushSettings = settings;
                 return Task.FromResult(0);
             }
 
             public void Start(PushRuntimeSettings limitations)
             {
-                Assert.AreEqual(10, limitations.MaxConcurrency);
+                if (pushSettings.InputQueue == Conventions.EndpointNamingConvention(typeof(ThrottledEndpoint)))
+                {
+                    // The LimitMessageProcessingConcurrencyTo setting only applies to the input queue
+                    Assert.AreEqual(10, limitations.MaxConcurrency);
```

Comment:
The benefit the to approach I described is that you'd be able to have the Assert in the expected location, instead of needing the comment telling you where to look for it.